docs/source/grpo_trainer.md

```diff
@@ -132,7 +132,8 @@ $$
 \text{clip}\left( \frac{\pi_\theta(o_{i,t} \mid q, o_{i,< t})}{\pi_{\theta_{\text{old}}}(o_{i,t} \mid q, o_{i,< t})}, 1 - \epsilon, 1 + \epsilon \right)
 $$
 A higher value means more tokens were affected by clipping, limiting how much the policy can change.
-
+- `mask_truncated_completions`: The completions that were truncated due to exceeding the maximum completion length. Therefore, excluded from the loss calculation. This feature was introduced as a metric of training stability in the (DAPO paper)[https://huggingface.co/papers/2503.14476]
```

Comment:
I don't think it's logged

Reply:
> > I don't think it's logged
> 
> 🤔 hmm, any comment on how to fix it ?

What do you want to log exactly? the number of masked completions? If so how is it different from `completions/clipped_ratio`?

Reply:
So that's something you forgot to remove, right?